app/tasks/schedule.php

```diff
@@ -0,0 +1,205 @@
+                    }
+                    $results = $dbForConsole->find('schedules', \array_merge($paginationQueries, [
+                        Query::equal('region', [App::getEnv('_APP_REGION')]),
+                        Query::equal('resourceType', ['function']),
+                        Query::greaterThan('resourceUpdatedAt', $lastUpdate),
+                    ]));
+
+                    $sum = count($results);
+                    $total = $total + $sum;
+                    foreach ($results as $document) {
+                        $org = isset($functions[$document['resourceId']]) ? strtotime($functions[$document['resourceId']]['resourceUpdatedAt']) : null;
+                        $new = strtotime($document['resourceUpdatedAt']);
+                        if ($document['active'] === false) {
+                            Console::warning("Removing:  {$document['resourceId']}");
+                            unset($functions[$document['resourceId']]);
+                        } elseif ($new > $org) {
+                            Console::warning("Updating:  {$document['resourceId']}");
+                            $functions[$document['resourceId']] =  [
+                                'resourceId' => $document->getAttribute('resourceId'),
+                                'schedule' => $document->getAttribute('schedule'),
+                                'resourceUpdatedAt' => $document->getAttribute('resourceUpdatedAt'),
+                            ];
+                        }
+                        $removeFromQueue($document['resourceId']);
+                    }
+
+                    $latestDocument = !empty(array_key_last($results)) ? $results[array_key_last($results)] : null;
+                }
+
+                $lastUpdate = DateTime::now();
+                $createQueue();
+                $timerEnd = \microtime(true);
+
+                Console::warning("Update timer: {$total} functions where updated in " . ($timerEnd - $timerStart) . " seconds");
+            });
+
+            /**
+             * The timer sends to worker every 1 min and re-enqueue matched functions.
+             */
+            Timer::tick(FUNCTION_ENQUEUE_TIMER * 1000, function () use ($dbForConsole, &$functions, &$queue) {
+                $timerStart = \microtime(true);
+                $time = DateTime::now();
+                $timeFrame =  DateTime::addSeconds(new \DateTime(), ENQUEUE_TIME_FRAME); /** 5 min */
+                $slot = (new \DateTime())->format('Y-m-d H:i:00.000');
+
+                Console::info("Enqueue proc started at: $time");
+
+                if (array_key_exists($slot, $queue)) {
+                    $schedule = $queue[$slot];
+                    console::info(count($schedule) . "  functions sent to worker  for time slot " . $slot);
+
+                    foreach ($schedule as $function) {
+                    /**
+                     * Enqueue function (here should be the Enqueue call
+                     */
```

Comment:
Can we fix the indentation of this comment ?

Reply:
sure